provide popover frame through props

roubkar opened the pull request.

jasonkuhrt commented:
@roubkar Hey, thanks for the contribution. But I'm a little lost about exactly what you're trying to do. Can you clarify? Thanks!

roubkar commented:
@jasonkuhrt Hey, sorry for the mess, Firstly I added ability to pass a reference for element which should be popover-s parent frame, last two commits are for React 16 compatibility, but I removed the parent frame logic during last commits. Sorry for taking your time.
